src/apps/Organization/Organization.js

```diff
@@ -0,0 +1,269 @@
+          </p>
+          {checksummedDaoAddr && (
+            <React.Fragment>
+              <div
+                css={`
+                  margin-top: ${2 * GU}px;
+                  margin-bottom: ${3 * GU}px;
+                `}
+              >
+                <LocalIdentityBadge
+                  entity={checksummedDaoAddr}
+                  shorten={shortAddresses}
+                />
+              </div>
+              <Info>
+                <strong css="font-weight: 800">
+                  Do not send ETH or ERC20 tokens to this address.
+                </strong>{' '}
+                {depositFundsHelpText}
+              </Info>
+            </React.Fragment>
+          )}
+        </Section>
+        {hasFinanceApp && testTokensEnabled(network.type) && (
+          <Section heading="Request test tokens">
+            <p
+              css={`
+                margin-bottom: ${2 * GU}px;
+                ${textStyle('body2')}
+              `}
+            >
+              Deposit some tokens into your organization for testing purposes.
+            </p>
+            <Button
+              label="Request test tokens"
+              icon={<IconCoin />}
+              display="all"
+              onClick={handleDepositTestTokens}
+              disabled={!enableTransactions}
+              css={`
+                margin-bottom: ${2 * GU}px;
+              `}
+            />
+            {enableTransactions ? (
+              <Info>
+                <p>
+                  Requesting tokens will assign random{' '}
+                  <strong css="font-weight: 800">test tokens</strong> to your
+                  organization. These tokens are named after existing projects,
+                  but keep in mind{' '}
+                  <strong css="font-weight: 800">
+                    they are not real tokens
+                  </strong>
+                  .
+                </p>
+                <p
+                  css={`
+                    margin-top: ${1 * GU}px;
+                  `}
+                >
+                  You can view the received tokens in{' '}
+                  <OpenAppButton onClick={handleOpenFinanceApp}>
+                    Finance
+                  </OpenAppButton>
+                  .
+                </p>
+              </Info>
+            ) : (
+              <Info mode="warning">
+                {walletNetwork !== network.type
+                  ? `Please select the ${sanitizeNetworkType(
+                      network.type
+                    )} network in your Ethereum provider.`
+                  : `Please unlock your account in your Ethereum provider.`}
+              </Info>
+            )}
+          </Section>
+        )}
+        {appsLoading ? (
+          <Section heading="Installed Aragon apps">
+            <div
+              css={`
+                display: flex;
+                align-items: center;
+                justify-content: center;
+                height: 180px;
```

Comment:
Not exactly `180px` but I don’t think it matters here:
```suggestion
                height: ${22 * GU}px;
```